OBSDOCS-3550: Release notes for 3.10.1 (#114186)
openshift/openshift-docs: modules/distr-tracing-rn-3.10.1.adoc

// Module included in the following assemblies:
//
// * release-notes/distr-tracing-rn.adoc

:_mod-docs-content-type: REFERENCE
[id="distr-tracing-rn-3-10-1_{context}"]
= Release notes for {dt} 3.10.1

[role="_abstract"]

> {Dit} 3.10.1 is provided through the link:https://catalog.redhat.com/software/containers/rhosdt/tempo-operator-bundle/642c3e0eacf1b5bdbba7654a/history[{TempoOperator} 0.21.0] and is based on the open source link:https://grafana.com/oss/tempo/[Grafana Tempo] 2.10.5.

Reviewer: ```suggestion
{Dit} 3.10.1 is provided through the link:https://catalog.redhat.com/software/containers/rhosdt/tempo-operator-bundle/642c3e0eacf1b5bdbba7654a/history[{TempoOperator} 0.21.0] and is based on the open source link:https://grafana.com/oss/tempo/[Grafana Tempo] 2.10.7.
```

[NOTE]
====
You can access some linked Jira tickets only with Red{nbsp}Hat credentials.
====

// The {dt} 3.10.1 release adds these features and enhancements:

// The {dt} 3.10.1 release adds the following Technology Preview features:
////
[IMPORTANT]
====
[subs="attributes+"]
Technology Preview features are not supported with Red{nbsp}Hat production service level agreements (SLAs) and might not be functionally complete. Red{nbsp}Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.

For more information about the support scope of Red{nbsp}Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
====
////
// :FeatureName: Each of these features
// include::snippets/technology-preview.adoc[leveloffset=+1]

// The {dt} 3.10.1 release deprecates the following features:

// The {dt} 3.10.1 release removes the following features:

// The {dt} 3.10.1 release has the following known issue:

> 

Author: @andreasgerstmayr, could you or Ruben review this file from this line to the end of it?

The {dt} 3.10.1 release fixes these issues:

Gateway query RBAC no longer fails to decode zstd-encoded requests::

> Before this update, the gateway query RBAC feature could not decode zstd-encoded requests. As a consequence, the Tracing UI in the OpenShift console failed with an *Error 502 Bad Gateway* message. With this update, the gateway query RBAC feature correctly decodes zstd-encoded requests. As a result, the Tracing UI works as expected.

Reviewer: ```suggestion
Gateway query RBAC no longer fails to decode zstd-encoded responses::
Before this update, the gateway query RBAC feature could not decode zstd-encoded responses from the Tempo API. As a consequence, the Tracing UI in the OpenShift console failed with an *Error 502 Bad Gateway* message. With this update, the gateway query RBAC feature correctly decodes zstd-encoded responses. As a result, the Tracing UI works as expected.
```

+

> link:https://redhat.atlassian.net/browse/TRACING-6438[TRACING-6438]

Reviewer: ```suggestion
link:https://redhat.atlassian.net/browse/TRACING-6419[TRACING-6419]
```

Reviewer: the above describes ticket TRACING-6438, RN for the https://redhat.atlassian.net/browse/TRACING-6438 issue are missing
